src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form-field {
+  margin: 20px 0;
+}
+
+.input {
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  width: 100%;
+  border-radius: 4px;
+}
+
+.big-search-bar {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px 25px;
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  font-size: 16px;
+}
+
+.small-search-bar {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+}
+
+.big-search-bar:hover {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big-search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px 25px;
+  font-weight: bold;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
+
+.small-search-bar:hover {
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small-search-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  font-weight: bold;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+}
```

Comment:
Don't repeat yourself. You have a lot of duplicated properties. You can move it to common class and use this class

Reply:
Fix here and in a whole file